src/web/index.js

```diff
@@ -30,9 +30,8 @@ const clipboard = new Clipboard(".clipboardButton");
 const addonUrl = process.env.ADDON_URL;
 
 // Period after which app loading indicator will disappear if add-on not found
-// If dev show right away to make debugging faster
-const LOADER_DELAY_PERIOD =
-  process.env.NODE_ENV === "development" ? 0 : 2000;
+// Default (2000) found in webpack.common.js
+const LOADER_DELAY_PERIOD = process.env.LOADER_DELAY_PERIOD;
```

Comment:
Does this need to be cast to a number? IIRC, it'd be a string if it was passed via the command line, and it's a string below in ./webpack.common.js:28. Unless `setTimeout()` auto casts...